### ISSUE TITLE
Completely Disable or Delete "Current Page Content" 

### ISSUE THREAD

**Issue by coreypotter** [netrivet/prophoto-issues#815]
I know that this can be disabled for all devices, but it still shows up in the code. Would love to at least have the option to delete the current page content block from templates or have a setting to disable it instead of hide it with CSS.

**Comment by lalunecreative** [comment]
You don't have to delete it with CSS. Just go into the block and click all the option to turn off visibility for each device. That's what we do when we don't need it.

**Comment by coreypotter** [comment]
Yes, but behind the scenes this is applying a display:none for all breakpoints. I'd like to see the code omitted completely. The primary reason is because the H1 with the pages title still gets added to the code, it is just hidden. While this is admittedly minor from an SEO perspective in almost all cases... there are cases where it could cause a conflict if people have the wrong thing in the title field, or haven't hidden it on all devices (by mistake). We have to remember that most end users aren't designers or developers and don't know what to watch out for. I've had a few SEO clients where I've had to amend issues related to the main content block that could actually cause problems with Google. 

I'm not sure of the exact reasoning for making this block impossible to delete, but I assume it has something to do with initiating the loop. Could be way off base on that, but I'd love to see the option to remove it completely from templates!

**Comment by lalunecreative** [comment]
It's because ProPhoto doesn't want to disable the ability to remove the Wordpress core content from the customizer. We had a chat about it initially in development before 6 came out. I'm not sure if they want to change stance on that or not. I can see the issue though, even if it's small for SEO there are still a lot of things they need to do to improve SEO performance, they could definitely probably amend this in some say so it would at least get better for SEO purposes while hidden. I suggested this one not that long ago - https://github.com/netrivet/prophoto-issues/issues/792 which would help us greatly, because we disable page titles on our pages most of the time, even in some of our designs. From a design standpoint they stick out like a sore thumb, and we rely on top placed widgets to take place of the H1 tag when we turn off the title and this would remedy that problem.

I also have to explain that it's super easy to hide content in ProPhoto by doing that method of hiding content all the time, it's a most underrated feature. On our own site it's how we easily enable/disable a menu bar with sales information in it when we run a sale. It's really great for things like that, or similar content you only want showing up at certain times, I'm hoping one day in the future we get a scheduler in there so that we can schedule when a block gets activated/deactivated. We haven't really asked for that, because I know it would be for fringe cases like the example I gave that we use it for.

**Comment by coreypotter** [comment]
Another time when this has been a problem for me is when I'm helping someone who updated from older versions of Prophoto or another theme where they had a ton of content in the main WordPress editor, and then instead of moving this content, they just hide the content block in the customizer and duplicate the content there. 

Some people even go as far as leaving the content in the WordPress editor because they want to get green lights on Yoast (sounds crazy, but I've heard it on multiple occasions). Having all of this duplicated in the code can slow down the load time even if it is hidden, especially if that content includes photos. 

I agree that the hiding is a great feature, and I love that there are 4 breakpoints where most builders only include 3.
